src/app/component/home/components/eco-events/eco-events-item/eco-events-item.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { NewsDto } from '../../../models/NewsDto';
+
+@Component({
+  selector: 'app-eco-events-item',
+  templateUrl: './eco-events-item.component.html',
+  styleUrls: ['./eco-events-item.component.scss']
+})
+export class EcoEventsItemComponent implements OnInit {
+  @Input() public ecoEvent: NewsDto;
+  @Input() public mainEvent = false;
+  readonly arrow = 'assets/img/icon/arrow.png';
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```

Comment:
empty methods and consequently - extra `import` and `implements`

Reply:
Fixed